zapcore/sampler.go

```diff
@@ -65,7 +65,13 @@ func fnv32a(s string) uint32 {
 	return hash
 }
 
-// Sample creates a facility that samples incoming entries.
+// Sample creates a Facility that samples incoming entries, which caps the CPU
+// and I/O load of logging while preserving a representative subset of your
+// logs.
```

Comment:
Sure. I was saving that until the sampler fix lands, but this is a perfectly fine time to add it.